bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
Yes, but I thought those might contain `version:`, but the development ones don't need it. Anyway, I just tried it and it doesn't work. Not including `version:` results in version `~unknown` pinned, not `dev`. Thanks.